Kickstarter-iOS/Views/Controllers/AddNewCardViewController.swift

```diff
@@ -203,4 +239,9 @@ STPPaymentCardTextFieldDelegate, MessageBannerViewControllerPresenting {
       }
     }
   }
+
+  private func cardBrandIsSupported(brand: STPCardBrand,
```

Comment:
This function should fit on one line if it wasn't intentional to split attributes on separate lines..

Reply:
I usually prefer to do that for long function declerations because it makes it more readable. It's personal preference though.